src/api/libcellml/units.h

```diff
@@ -426,18 +426,35 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
     bool requiresImports() const;
 
     /**
+     * @overload
+     * 
      * @brief Return the scaling factor difference between two @c Units.
      *
-     * This can be interpreted as `factor`, where units2 = factor*units1.  If the units
-     * are not compatible the factor returned is 0.0.
+     * This can be interpreted as `factor`, where units2 = factor*units1.  If the units are not 
+     * compatible the factor returned is 0.0.
      *
      * @param units1 The first units to compare.
-     * @param units2 The second units to compare.
+     * @param units2 The second units to compare. 
      *
-     * @return The factor units2/units1, where the units are compatible, or 0.0 otherwise.
+     * @return The factor units2/units1.  Where the units are incompatible the factor returned is 0.0.
      */
     static double scalingFactor(const UnitsPtr &units1, const UnitsPtr &units2);
```

Comment:
I think a default argument would be sufficient, I don't see a need for a separate function.

```
static double scalingFactor(const UnitsPtr &units1, const UnitsPtr &units2, bool checkCompatibility = true);
```

I don't think there is any point passing a bool by reference either.

Reply:
Agreed.

Reply:
Done.